backend/tests/integration/pidstore/minters/test_minters_doi.py

```diff
@@ -108,3 +108,12 @@ def test_mitner_dois_missing_schema(base_app, db, es, create_record_factory):
 
     with pytest.raises(MissingSchema):
         DoiMinter.mint(record_id, record_data)
+
+
+def test_new_doi_converted_to_lowercase(base_app, db, create_record):
+    expected_doi = "10.1109/tpel.2019.2900393"
+    doi_value = "10.1109/TPEL.2019.2900393"
+    data = {"dois": [{"value": doi_value}]}
+    create_record("lit", data=data)
+    doi = PersistentIdentifier.query.filter_by(pid_type="doi").first()
+    assert expected_doi == doi.pid_value
```

Comment:
can you also add a test where you create a record with the same DOI twice in the record, but each time with different case? this might happen, and I'm not sure it's handled correctly.